src/main/resources/study.sql

```diff
@@ -1,94 +0,0 @@
-
```

Comment:
Why do you remove this file ? It is referenced in the deployment repository README.md file.

Reply:
its accitental